.github/workflows/test.yaml

```diff
@@ -85,7 +85,7 @@ jobs:
         with:
           base: 'specs/base.yaml'
           revision: 'specs/revision-breaking.yaml'
-          fail-on-diff: false
+          fail-on: 'WARN'
           output-to-file: 'breaking.txt'
```

Comment:
Why this test do not fail?
[oasdiff docs](https://github.com/Tufin/oasdiff/blob/main/BREAKING-CHANGES.md):
> To exit with return code 1 when any breaking changes are found, WARN-level or ERR-level, add the --fail-on WARN flag.

Can you please explain the why you decided to change `fail-on-diff: false` to `fail-on: 'WARN'`?